### PR TITLE
fix(themes): update highlight token for dark themes

### LINKED ISSUE #3182: [Theme] Highlight token value doesn't work in dark themes
## Token value

Dark themes `$highlight` has wrong values. Currently using `blue-90` on both Gray 90 and Gray 100 themes. 

Current values for `$highlight` token: 

| White | Gray 10 | Gray 90 | Gray 100 |
| --- | --- | --- | --- |
| `blue-20` | `blue-20` | `blue-90` | `blue-90` |

Should be changed to below values for `$highlight` token: 

| White | Gray 10 | Gray 90 | Gray 100 |
| --- | --- | --- | --- |
| `blue-20` | `blue-20` | `blue-70` | `blue-80` |

image

### Related bug

We never caught this bug because our dark theme Date picker is not using dark theme values – issue made https://github.com/carbon-design-system/carbon/issues/3183

image

### PR DESCRIPTION
**Pull request by jendowns**
Closes #3182 

#### Changelog

**Changed**

- changed the `$highlight` token colors for dark themes, per https://github.com/carbon-design-system/carbon/issues/3182#issue-461052157
- updated vendor files in monorepo

### CODE REVIEW

**Review by shixiedesign — APPROVED** [review]
Looks great! Thanks for fixing `$hover-primary-text` as well! 🎉🎉🎉

**Review by abbeyhrt — APPROVED** [review]
Once the tests pass, this is good to go! 🎉

**Comment by jendowns** [comment]
@abbeyhrt @shixiedesign @emyarod Are any of you able to restart the CI builds for me? Seems like a time out situation? I can't see details on my end except `failed during stage 'building site': Build script returned non-zero exit code: 1` etc 🤔

**Comment by tw15egan** [comment]
@jendowns triggered a rebuild, let's see what happens 🤞

**Review by tw15egan — APPROVED** [review]
LGTM once CircleCI passes 👍

File: packages/themes/src/g90.js [hunk 1]
```diff
@@ -113,7 +112,7 @@ export const disabled01 = gray80;
 export const disabled02 = gray70;
 export const disabled03 = gray50;
 
-export const highlight = blue90;
+export const highlight = blue70;
```
Comment (tw15egan) [inline packages/themes/src/g90.js:116]: Looks like the test is failing because we're referencing `blue70` here but not importing it

**Comment by jendowns** [comment]
Thank you @tw15egan! Looks like the tests are passing now 🙌

**Pull request merged**